stdlib/src/collections/list.mojo

```diff
@@ -666,3 +656,9 @@ struct List[T: CollectionElement](CollectionElement, Sized, Boolable):
                 result += ", "
         result += "]"
         return result
+
+    @always_inline
+    fn _normalize_index(self, index: Int) -> Int:
+        if index < 0:
```

Comment:
Suggestion: invert the condition to reduce indentation. So 

```suggestion
        if index >= 0:
           return index 
        return complicated expr 
```

Reply:
The indentation would remain the same @abduld after your suggestion